### PR TITLE
Fixes #1664Setting Up Backend

### LINKED ISSUE #1664: Update Installation Docs for Backend
**I'm submitting a ...**
- [X] bug report
- [ ] feature request

Please update the Installation Docs for following:
- [ ] Generic Installation Instructions. Please see [this](https://github.com/fossasia/open-event-server/blob/development/docs/installation/basic.md) for reference.
- [ ] Installation on Linux/Unix using Virtualenv. Please see [this](https://github.com/fossasia/open-event-server/blob/development/docs/installation/local.md) for reference. 
- [ ] Installaion on Windows or Vagrant Installation. Please see [this](https://github.com/fossasia/open-event-server/blob/development/docs/installation/vagrant.md) for reference.
- [ ] Deployment on Heroku. Please see [this](https://github.com/fossasia/open-event-server/blob/development/docs/installation/heroku.md) for reference.
- [ ] Deployment on Docker. Please see [this](https://github.com/fossasia/open-event-server/blob/development/docs/installation/docker.md) for reference.

1. Please **open separate Issue** for respective Installation docs.
2. Installation docs should be placed [`api/docs/installation`](https://github.com/fossasia/badgeyay/tree/development/api/docs/installation).
3. Please update the link of installation docs in the [`README`](https://github.com/fossasia/badgeyay/blob/development/api/README.md)
3. Every Installation doc should contain a good quality screenshot for every step mentioned.
4. Use Github `Markdown` uniformly for formatting the Installation steps.

### PR DESCRIPTION
**Pull request by sk9331657**
**Fixes #1664 Local Environment Setup ** 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-  Setting up backend database and removing unwanted steps

### CODE REVIEW

File: api/docs/installation/localvir.md [hunk 1]
```diff
@@ -13,8 +13,9 @@ $ sudo apt-get update
 Make sure you have the dependencies mentioned above installed before proceeding further.
 For a start, fork BadgeYay to your own github account. Then, clone it to your local system.
 
+* **Step 1**
 ```sh
-$ git clone -b development https://github.com/<your_username>/badgeyay.git
+$ git clone -b development https://github.com//badgeyay.git
```
Comment (gabru-md) [inline api/docs/installation/localvir.md]: why have you changed this line ? :sweat_smile:

Comment (sk9331657) [reply api/docs/installation/localvir.md]: Oh That is by mistake , I will resolve it.

**Comment by sk9331657** [comment]
@gabru-md  Please review it.

**Comment by sk9331657** [comment]
@gabru-md  @ParthS007  Please approve this pull request and merge this  patch .

**Comment by ParthS007** [comment]
@sk9331657 Please resolve conflicts. :+1:

**Comment by gabru-md** [comment]
@sk9331657 can you please update your PR ? there seems to be some conflicts.

**Comment by sk9331657** [comment]
@gabru-md @ParthS007 I am removed the conflicts. It is now ready to be merged.

**Pull request merged**